index.bs

```diff
@@ -1372,7 +1372,7 @@ To <dfn>deserialize local value</dfn> given |realm| and |local protocol value|:
 
       1. Let |date result| be [=Construct=]([=Date=], |value|).
 
-      1. Assert: [=ReturnIfAbrupt=](|date result|) is not a [=abrupt completion=].
+      1. Assert: [=ReturnIfAbrupt=](|date result|) is not an [=abrupt completion=].
```

Comment:
```suggestion
      1. Assert: |date result| is not an [=abrupt completion=].
```

Apologies for the confusion, I didn't mean that you need to call `ReturnIfAbrupt`, I just meant that ES is already calling that so the return type is either a value or an abrupt completion.